### PR TITLE
Update Keras model example and docs

### PR DESCRIPTION
**Pull request by tedhtchang**
#### Motivation
Update Keras doc and examples to reflect the changes in the updated minio images

### CODE REVIEW

File: docs/model-types/README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 By leveraging existing third-party model servers, we support a number of standard ML model formats out-of-the box, with more to follow. Currently supported model types:
 
+- [keras] (keras.md)
 - [LightGBM](lightgbm.md)
```
Comment (chinhuang007) [inline docs/model-types/README.md:6]: Maybe remove the blank character between "]" and "("?

File: config/example-predictors/example-keras-mnist-predictor.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 IBM Corporation
```
Comment (pvaneck) [inline config/example-predictors/example-keras-mnist-predictor.yaml:1]: 2021 -> 2022

File: docs/model-types/keras.md [hunk 3]
```diff
@@ -35,7 +35,7 @@ metadata:
 spec:
   modelType:
     name: keras
-  path: tensorflow-model/keras-mnist/mnist.h5
+  path: keras-models/mnist.h5
```
Comment (pvaneck) [inline docs/model-types/keras.md:38]: This doesn't match the path above. Which is the correct one?

File: docs/model-types/README.md (hunk 1 above)
Comment (pvaneck) [reply docs/model-types/README.md:6]: Right, also, a nit: capitalize the K in keras.

**Review by pvaneck — CHANGES_REQUESTED** [review]

**Review by pvaneck — APPROVED** [review]

**Pull request merged**